dockerfiles/PySparkContainerDockerfile

```diff
@@ -6,6 +6,7 @@ FROM ${REGISTRY}/${RPC_VERSION}-rpc:${CODE_VERSION}
 LABEL maintainer="Dan Crankshaw <dscrankshaw@gmail.com>"
 
 RUN mkdir -p /usr/share/man/man1 && \
+    apt-get update -qq -y && \
```

Comment:
This should be fixed in the other PR as well right?

Reply:
Yes. This code was already merged with `develop` branch by https://github.com/ucbrise/clipper/pull/659.